packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -527,29 +527,33 @@ export interface DocsQAProps {
   corpus: Corpus;
 
   /**
-   * The query to answer.
+   * The question to answer.
    */
   question: string;
 
   /**
    *
-   * The maximum number of documents to return.
+   * When processing a DocsQA query, the most relevent chunks are presented to the model. This
+   * field limits the number of chunks to consider. If this value is too large, the size
+   * of the chunks may exceed the token limit of the model.
```

Comment:
I think this and all our docs need to walk a tightrope of "explain concisely to someone who already knows what a `chunk` is" and "provide an on-ramp to people brand new to LLMs who don't." For the latter group, we could link to https://docs.ai-jsx.com/guides/docsqa. (Or maybe we need a glossary.)